### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by egichuhis**
This PR contains HTML&CSS linter configuration, an HTML file and a CSS file

### CODE REVIEW

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @egichuhis    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the Review.

- [x] kindly make sure that when you Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) "[.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc)" and "[.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json)" present in your Repository.
Your Repository Look likes this :  

c1

But [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) Look likes this : 

caz


## Optional suggestions

- [ ] Kindly fix the [MIT license](https://choosealicense.com/licenses/mit/) in your Readme file, The Link is broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# testing
+test.md
+
+
+
```
[the remark this answers is not in the document]
Comment (egichuhis) [reply .gitignore:5]: Hello @Abenezer-Tilahun,
Thank you for the thorough review. I am working on the fixes and updates and I will finish by tomorrow. 
Cheers!

**Comment by egichuhis** [comment]
Hello @Abenezer-Tilahun,

I have made the necessary corrections. Thank you for taking the time to review my work.
Cheers!

**Review by jerryowusu — APPROVED** [review]
Hi @egichuhis , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clapping-leonardo-dicaprio

## Highlights

-  No linter errors. :heavy_check_mark: 
- You used correct GitHub flow :heavy_check_mark: 
- Good and meaningful commit messages :heavy_check_mark: 
- All linters passing :heavy_check_mark: 

## Optional suggestions
 - [ ] The link to the license is still broken. Kindly fix that. You can consider fixing that by creating a file named "LICENSE" (in the same location as the "README.md" file), then copy and paste the content of this [link](https://choosealicense.com/licenses/mit/) and edit the name and year accordingly.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+
+- GitHub: [@githubhandle](https://github.com/egichuhis)
+- Twitter: [@twitterhandle](https://twitter.com/egichuhis)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/edwin-gichuhi/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
[the remark this answers is not in the document]
Comment (egichuhis) [reply README.md]: Thank you @jerryowusu for the thoroughness in your review. I hope to emulate your stewardship and industry. I have fixed the optional issues above. 
Cheers!

**Pull request merged**